src/site/apt/index.apt.vm

```diff
@@ -52,7 +52,7 @@ ${project.name}
   already contain the answer to your question as part of an older thread. Hence, it is also worth browsing/searching
   the {{{./mailing-lists.html}mail archive}}.
 
-  If you feel like the plugin is missing a feature or has a defect, you can fill a feature request or bug report in our
+  If you feel like the plugin is missing a feature or has a defect, you can file a feature request or bug report in our
```

Comment:
submit ?

Reply:
as soon as someone approves

Reply:
> submit ?

I meant as another suggestion for the `fill` / `file` wording.

Reply:
Oh, I get it now. Yes, that is better wording.